corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -128,26 +131,28 @@
     {% endblocktrans %}
 </div>
 {% endif %}
-<div class="pull-right">
-    {% if auth_context.authenticated %}
-        <i class="icon-lock"></i>
-        {% if user_info.id != auth_user_info.id %}
-        {% blocktrans with auth_user=auth_user_info|pretty_doc_info user=user_info|pretty_doc_info %}
-            Submitted by {{ auth_user }} on behalf of {{ user }}
-        {% endblocktrans %}
+<div id="submit-info">
+    <div class="pull-right">
```

Comment:
Move this element after `<ul class="nav nav-tabs clearfix">`. It shouldn't be above the tabs in the first place. Also, `clearfix` on the `nav-tabs` element isn't really doing much here. I would just get rid of the surrounding `#submit-info` div too, tbh.